ui/pages/permissions-connect/flask/util.js

```diff
@@ -17,7 +17,9 @@ export function getSnapInstallWarnings(permissions, targetSubjectMetadata, t) {
   return [
     ...flatMap(bip32EntropyPermissions, (permission, i) =>
       permission.caveats[0].value.map(({ path, curve }) => ({
-        id: `key-access-bip32-${path.join('/')}-${curve}-${i}`,
+        id: `key-access-bip32-${path
+          .join('-')
+          .replace(/'/gu, 'h')}-${curve}-${i}`,
```

Comment:
```suggestion
          .replaceAll(`'`, 'h')}-${curve}-${i}`,
```

Reply:
https://caniuse.com/?search=replaceAll - we cant use that.